documentation/TESTING_PLAN.md

```diff
@@ -16,7 +16,16 @@ In general, we're approaching testing bottom-up, with each unit function tested
 
 ### Unit-level tests
 
-- TBD 10
+- Test that impassible walls are impassible by the player.
+- Test that the level is complete/congratulatory message when and only when all boxes are moved to designated goal cells.
+- Test when a level is complete, the user is prompted to move to the next level or jump to a different level.
```

Comment:
Same as above -- prompts are UI concepts that the game engine doesn't care about.